strapi/strapi
# feat(cloud-cli): preselect node-version based on local node-version

### What does it do?

Adding a preselection on the currently running NodeJS version while selecting the Node Version on project creation in Cloud CLI.

### Why is it needed?

Better UX

### How to test it?

- Try to deploy a new project to the Cloud using `strapi deploy` command.
- Setup your local to either run on Node 18 or 20
- See the node version preselected when asked to select a node version

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
